pkg/servicecentral/metadata.go

```diff
@@ -29,6 +32,21 @@ func SetPagerDutyMetadata(serviceCentralData *ServiceData, m *creator_v1.PagerDu
 	return setMetadata(serviceCentralData, PagerDutyMetadataKey, m)
 }
 
+// GetOpsGenieMetadata reads the opsgenie metadata out of a service
+func GetOpsGenieMetadata(serviceCentralData *ServiceData) (*creator_v1.OpsGenieMetadata, error) {
+	var m creator_v1.OpsGenieMetadata
+	found, err := unmarshalFromMiscData(serviceCentralData, OpsGenieMetadataKey, &m)
```

Comment:
This is wrong, we should pull team name from service attribute and construct OpsGenieMetadata struct on the fly (no unmarshalling).